app/assets/stylesheets/custom.scss

```diff
@@ -128,16 +112,53 @@ a {
 
 #store-name {
   color: #504d4f;
+  float: left;
+  font-size: 1.6em;
+  margin: auto;
 }
 
 .retire-btn {
   margin-bottom: 54px;
 }
 
+#complete-your-loan {
+  text-align: center;
+}
+
 td {
   background-color: rgba(255, 255, 255, .5);
 }
 
+// flash alerts
+#flash_error {
+  text-align: center;
+  color: red;
+}
+
+#flash_alert {
+  color:red;
+  text-align: center;
+}
+
+#flash_notice {
+  text-align: center;
+  color:green;
```

Comment:
Color `green` should be written in hexadecimal form as `#008000`<br>Colon after property should be followed by one space